sql/core/benchmarks/SubExprEliminationBenchmark-results.txt

```diff
@@ -3,23 +3,23 @@ Benchmark for performance of subexpression elimination
 ================================================================================================
 
 Preparing data for benchmarking ...
-OpenJDK 64-Bit Server VM 1.8.0_362-b09 on Linux 5.15.0-1031-azure
-Intel(R) Xeon(R) Platinum 8370C CPU @ 2.80GHz
+OpenJDK 64-Bit Server VM 17.0.8+7-LTS on Linux 5.15.0-1046-azure
+Intel(R) Xeon(R) Platinum 8272CL CPU @ 2.60GHz
 from_json as subExpr in Project:          Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-subExprElimination false, codegen: true            6737           6973         320          0.0    67369399.0       1.0X
-subExprElimination false, codegen: false           6271           6373          88          0.0    62710705.3       1.1X
-subExprElimination true, codegen: true              867            870           3          0.0     8666868.2       7.8X
-subExprElimination true, codegen: false             811            824          23          0.0     8107336.6       8.3X
+subExprElimination false, codegen: true            8437           8618         196          0.0    84373053.3       1.0X
+subExprElimination false, codegen: false           8291           8425         227          0.0    82910513.2       1.0X
+subExprElimination true, codegen: true             2180           2231          45          0.0    21795155.9       3.9X
```

Comment:
`subExprElimination true, codegen: true`  and `subExprElimination true, codegen: false` have also significantly slowed down.